Tools/wasm/wasi.py

```diff
@@ -270,6 +270,10 @@ def make_wasi_python(context, working_dir):
 
     exec_script = working_dir / "python.sh"
     subprocess.check_call([exec_script, "--version"])
+    print(
+        f"🎉 Use '{exec_script.relative_to(context.init_dir)}' "
```

Comment:
```suggestion
        f"Use '{exec_script.relative_to(context.init_dir)}' "
```
Do we *really* want emojis in terminals (sure, it may be getting better, but still quite a lot of platforms with poor support)?

Reply:
Yes. No one has ever complained about them in this script and it makes the output easier to read/skim.